docs/docs/reference-react.md

```diff
@@ -1,24 +1,67 @@
 ---
-id: reference-react
+id: react-api
 title: React Top-Level API
 layout: docs
 category: Reference
-permalink: docs/reference-react.html
+permalink: docs/react-api.html
 redirect_from: "/docs/reference.html"
 ---
 
 `React` is the entry point to the React library.
 
-## Reference
+## Overview
+
+### Components
+
+React components let you split the UI into independent, reusable pieces, and think about each piece in isolation. React components can be defined by subclassing `React.Component`.
 
  - [`React.Component`](#react.component)
- - [`createClass()`](#createclass)
```

Comment:
Maybe note the separation here too? "If you don't use ES6 classes, you may use this helper instead" or something. Right now they appear grouped together as if they are equally useful.